ui/js/main.js

```diff
@@ -130,4 +133,40 @@ var init = function() {
   }
 };
 
+const download = (url, dest, lang, cb) => {
```

Comment:
I'd like this to follow more of the standard React/Redux model that this rest of the app is using. You can model the existing settings logic (`page/settings/view.jsx`, `page/settings/index.js`, `selectors/settings.js`, `reducers/settings.js`, `selectors/settings.js`). If this isn't making sense, chat with me.